### ISSUE TITLE
Use new Icon API for Tabledrag multi-arrow icons

### ISSUE THREAD

**Issue by quicksketch** [backdrop/backdrop-issues#6493]
Now that we have the new SVG icons from https://github.com/backdrop/backdrop-issues/issues/364, we should use one for tabledrag:

![image](https://github.com/backdrop/backdrop-issues/assets/100206/3dd7338e-bb76-4746-9fc0-97c429e22dc9)

**Comment by quicksketch** [comment]
Work-in-progress PR filed at https://github.com/backdrop/backdrop/pull/4720

![image](https://github.com/backdrop/backdrop-issues/assets/100206/918808fb-6905-4e57-b684-c89936740f32)

Although not _strictly_ necessary, I added `tabledrag.js` to a library so it can be added (along with its icons) via `backdrop_add_library()` instead of `backdrop_add_js()`. I also moved its CSS outside of `system.css`, since it is not needed on most pages.

**Comment by indigoxela** [comment]
> I added tabledrag.js to a library ...  I also moved its CSS outside of system.css

Ouch, that doesn't sound like it's backwards compatible move, though. At least one of my contrib modules and probably most of my themes will be affected by such a change. :grimacing:

**Comment by argiepiano** [comment]
AFAIK you can still load `tabledrag.js` the "old" way with backdrop_add_js, despite being added to a library, no?

**Comment by quicksketch** [comment]
@indigoxela Yeah... I wonder if this is feasible as well; moving tabledrag CSS into its own file. Are you actually overriding `system.css` (not `system.theme.css`)? That's a big set of functional CSS to override.

@argiepiano From what I've seen in contrib modules, it's pretty unusual to add tabledrag.js directly, usually you call `backdrop_add_tabledrag()`, which is needed to add the corresponding settings to the page. That in turn adds the JS (or the new library in this case).

**Comment by indigoxela** [comment]
> Are you actually overriding system.css (not system.theme.css)?

I'm overriding tabledrag styles in (almost?) every theme I create. Just CSS. But moving things around sounds really dangerous re backwards compatibility. Needs proper testing, as tabledrag's used in lots of module and probably CSS gets overridden in many (most) themes. Contrib or custom.

> ... usually you call backdrop_add_tabledrag()

Yes, that's what my module uses.

However, tabledrag is one of those things used in lots of modules. So my by bc alarm got triggered. :wink:

**Comment by klonos** [comment]
With regards to the icons:
- consider increasing the contrast, as that light gray over white background seems insufficient (haven't actually tested though)
- perhaps also use `arrow-elbow-down-left`/`arrow-elbow-down-right` icons from the Phosphor icon set instead of the custom `tree.png` and `tree-bottom.png` ones we currently use
